snuba/datasets/processors/querylog_processor.py

```diff
@@ -93,6 +91,7 @@ def __extract_query_list(
             groupby_columns.append(profile["groupby_cols"])
             array_join_columns.append(profile["array_join_cols"])
             bytes_scanned_columns.append(result_profile.get("bytes", 0))
+            duration_ms.append(result_profile.get("elapsed", 0))
```

Comment:
The result returned from clickhouse is in seconds while the column being written to is in milliseconds. https://clickhouse-driver.readthedocs.io/en/latest/features.html?highlight=elapsed#query-execution-statistics. You should convert it to milliseconds before writing to querylog

Reply:
Good catch. Does it make more sense to preserve the original nature of `ClickhouseProfile` and do the conversion in the querylog processor or before the message is produced to topic?

Reply:
What you did seems fine to me.